crates/ironrdp-connector/src/credssp_sequence.rs

```diff
@@ -0,0 +1,214 @@
+use ironrdp_pdu::write_buf::WriteBuf;
+use ironrdp_pdu::{nego, PduHint};
+use sspi::credssp::{self, ClientState, CredSspClient};
+use sspi::generator::{Generator, NetworkRequest};
+use sspi::negotiate::ProtocolConfig;
+use sspi::KerberosConfig as SSPIKerberosConfig;
```

Comment:
You can use `sspi::KerberosConfig` instead of renaming it

Reply:
updated